data/pages/transparency.mdx

```diff
@@ -47,6 +47,7 @@ as in the internal and external processes of our organization.
 - [2024](/docs/financials/2022-2024.pdf)
 - [2025-Q1](/docs/financials/2025-Q1-Q2.pdf)
 - [2025-Q2](/docs/financials/2025-Q1-Q2.pdf)
+- [2025-Q3](/docs/financials/2025-Q3.pdf)
```

Comment:
According to the preview, this pdf contains all 3 quarters Q1-Q2-Q3 for 2025. Might make more sense to drop the other two links (that are both to the same pdf) and have only one link for 2025 to this new pdf?